optax/_src/alias_test.py

```diff
@@ -142,6 +142,25 @@ def test_explicit_dtype(self, dtype):
     adam_state, _, _ = tx.init(jnp.array([0.0, 0.0]))
     self.assertEqual(expected_dtype, adam_state.mu.dtype)
 
+  def test_online_newton_step_decreases_loss(self):
+    """We check that the online newton step update decrease a loss function.
```

Comment:
typo: ... newton step update decreases a

But actually, the "We" isn't typical for test docstrings. I'd suggest:

```
"""Checks that the online newton step update decreases the loss."""
```